### PR TITLE
Add send link demo

### PR DESCRIPTION
**Pull request by linyimin-bupt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,10 @@
   },
   "homepage": "https://github.com/Chatie/wechaty-getting-started#readme",
   "dependencies": {
+    "@types/node": "^10.9.4",
```
Comment (huan) [inline package.json]: Please remove unnecessary changes in package.json

File: examples/professional/send-link.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  .on('error', async err => {
+    console.error('BOT ERROR:', `exit for ${JSON.stringify(err.message || '')}`)
+  })
+  .on('message', async function (m: Message) {
+    if (m.text() === 'ding') {
+      const link = new UrlLink({
+        description : '一个段子，虽然今天看起来有点老，但却仍然形象：你用小米手机，穿凡客T恤，泡贝塔咖啡听创业讲座，宅家看哈佛公开',
+        title       : 'SHE成团17年发布新歌《十七》MV 还原出道时经典造型(视频)',
+        url         : 'https://view.inews.qq.com/w/WXN201809080546120A0?refer=nwx&bat_id=1114015870&cur_pos=3&grp_type=region&rg=2&openid=o04IBAE_ggt-jQaC2fgkTQUjFWWs&groupid=1536393804&msgid=3',
+        thumbnailUrl: 'https://avatars2.githubusercontent.com/u/17776713?s=460&v=4',
```
Comment (lijiarui) [inline examples/professional/send-link.ts]: This content should change to an official standard content:

```ts
  const urlLink = new UrlLink ({
     description : 'WeChat Bot SDK for Individual Account, Powered by TypeScript, Docker, and Love',
     thumbnailUrl: 'https://avatars0.githubusercontent.com/u/25162437?s=200&v=4',
     title       : 'Welcome to Wechaty',
     url         : 'https://github.com/chatie/wechaty',
   })
```

**Comment by huan** [comment]
@linyimin-bupt Could you please follow the code reviews and fix the conflicting so that we can merge this PR?

Thanks!

**Comment by huan** [comment]
Ping @linyimin-bupt

**Comment by linyimin-bupt** [comment]
I can't find the original repository, so I create a new pr for fix these problems.

**Comment by linyimin-bupt** [comment]
If there are no any problem,could I close this pr?

**Comment by huan** [comment]
Great to hear from you again!

Please feel free to help yourself close this PR.
